Packaging.cmake

```diff
@@ -0,0 +1,64 @@
+if(NOT VERSION)
+	set(VERSION "unknown")
+endif()
```

Comment:
Is this part needed? `awesomeConfig.cmake` sets `VERSION` to `devel` if it doesn't find anything better.